code/modules/client/loadout/loadout_suit.dm

```diff
@@ -3,23 +3,173 @@
 	slot = SLOT_WEAR_SUIT
 	sort_category = "External Wear"
 
+//ALT ARMOR, HAZARD & MEDICAL VESTS, + LABCOATS
+
+/datum/gear/suit/vests/hazard_vest
+	display_name = "hazard vest"
+	path = /obj/item/clothing/suit/hazardvest
+	allowed_roles = list("Chief Engineer", "Station Engineer", "Atmospheric Technician", "Quartermaster", "Cargo Technician")
+	cost = 2500
+
+/datum/gear/suit/vests/armor/alt //just the alternate appearance armor vest that is in sec lockers.
+	display_name = "type I-B armor vest"
+	path = /obj/item/clothing/suit/armor/vest/alt
+	allowed_roles = list("Security Officer", "Warden", "Head of Security", "Head of Personnel")
+	cost = 5000
+
+/datum/gear/suit/vests/armor/blueshirt //about that beer I owe ya
```

Comment:
remove

Reply:
This is a reskin that can be purchased from the sec vendor, no armor changes.